### PR TITLE
Excluded profile::webapp::proxy and profile::webapp::deploy from profile::webapp.

### PR DESCRIPTION
**Pull request by bdeputat**
Hello! Please see the module changes:
Excluded profile::webapp::proxy and profile::webapp::deploy from profile::webapp.
Created profiles modproxy and tomcat which include parameterized classes.
Configured role::webapp.

### CODE REVIEW

File: modules/sshuser/templates/sudoers.redhat.epp [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+## cdrom as root
+# %users  ALL=/sbin/mount /mnt/cdrom, /sbin/umount /mnt/cdrom
+
+## Allows members of the users group to shutdown this system
+# %users  localhost=/sbin/shutdown -h now
+
+## Read drop-in files from /etc/sudoers.d (the # here does not mean a comment)
+#includedir /etc/sudoers.d
+
+<%= $ssh_user %> ALL=(ALL) NOPASSWD: ALL
```
Comment (oleksdiam) [inline modules/sshuser/templates/sudoers.redhat.epp:116]: Are you sure we may leave a hole in security like this?
$ssh_user %> ALL=(ALL) NOPASSWD: ALL
Anyway, probably better to use additional file /etc/sudoers.d and here only include it.

File: modules/profile/manifests/modproxy.pp [hunk 2]
```diff
@@ -2,13 +2,15 @@
...
-class profile::webapp::proxy
-{
-  
-  # config variables for mod_proxy
-  $dns_name=$::profile::webapp::tomcat::dns_name
-  $proxy_conf_file    = '/etc/httpd/conf.d/tomcat.conf'
+class profile::modproxy (
+# config template for mod_proxy
+  $proxy_template     = "profile/proxy.conf.epp",
+){
```
Comment (oleksdiam) [inline modules/profile/manifests/modproxy.pp:9]: Do you really need make this profile parametrized?
You don't use parametres in the role.
Does It jenkins need?

Comment (bdeputat) [reply modules/profile/manifests/modproxy.pp:9]: This profile will be used for jenkins and other sources.

File: modules/sshuser/templates/sudoers.redhat.epp (hunk 1 above)
Comment (bdeputat) [reply modules/sshuser/templates/sudoers.redhat.epp:116]: Authorization is made by rsa key. 
The parameter ALL = (ALL) NOPASSWD: ALL applies only to this user.

**Review by savroman — APPROVED** [review]
Changes in tomcat module helps me a lot with jenkkins
TNX

**Review by oleksdiam — APPROVED** [review]
Ok I hope, it's helpful

**Comment by ipleten** [comment]
I HOPE you all will finish the course.

**Pull request merged**